maths/sum_of_arithmetic_series.py

```diff
@@ -1,12 +1,21 @@
 # DarkCoder
-def sumOfSeries(a, d, n):
-    sum = ((n / 2) * (2 * a * (n - 1) * d))  # formula for sum of series
-    print(sum)
+def sum_of_series(first_term, common_difference, number_of_terms):
+    """
+    Find the sum of n terms in an arithmetic progression.
+
+    >>> sum_of_series(1, 1, 10)
+    55.0
+    >>> sum_of_series(1, 10, 100)
+    49600.0
+    """
+    sum = ((number_of_terms / 2) * (2 * first_term + (number_of_terms - 1) * common_difference))  # formula for sum of series
```

Comment:
Over quality much improved now. 
Thank you for your contribution.
Kindly fix this. The comment is bleeding beyond 88 chars. 
You might want to add the comment before the expression.